.cirrus.yml

```diff
@@ -0,0 +1,61 @@
+low_scale_task:
+
+  compute_engine_instance:
+    image_project: fedora-cloud
+    image: family/fedora-cloud-35
+    platform: linux
+    memory: 8G
+
+  env:
+    DEPENDENCIES: git ansible podman podman-docker
+
+  runtime_cache:
+    folder: runtime-cache
+
+  configure_ssh_script:
+    - mkdir -p /root/.ssh/
+    - ssh-keygen -t rsa -N '' -q -f /root/.ssh/id_rsa
+    - ssh-keyscan localhost >> /root/.ssh/known_hosts
+    - cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
+    - chmod og-wx /root/.ssh/authorized_keys
+    - ssh root@localhost -v echo Hello
+
+  install_dependencies_script:
+    - dnf install -y ${DEPENDENCIES}
+
+  configure_physical_deployment_script:
+    - rm -rf physical-deployments/physical-deployment.yml
+    - mv physical-deployments/localhost.yml
+         physical-deployments/physical-deployment.yml
```

Comment:
Nit: I'd change this script to:
```
- echo "PHYS_DEPLOYMENT=$(pwd)/physical-deployments/localhost.yml" >> $CIRRUS_ENV
```
in order to point do.sh to the right deployment.

Reply:
Yep, sounds good.